.taskcluster.yml

```diff
@@ -102,12 +102,23 @@ tasks:
             features:
               # Needed for access to secret
               taskclusterProxy: true
+              dind: true
             maxRunTime: 3600
-            image: mozilla/taskboot:latest
+            image: mozillareleases/python-test-runner
```

Comment:
Are we to use this image for all the *scripts that we'll migrate?

Reply:
I don't have a better candidate for now, but we probably want to use something more generic in the future. I'm going to revisit this approach very soon in any case.